app/_locales/en/messages.json

```diff
@@ -6815,5 +6815,11 @@
   },
   "zeroGasPriceOnSpeedUpError": {
     "message": "Zero gas price on speed up"
+  },
+  "smartTransactionsEnabledMessage": {
```

Comment:
nit: All keys in this file are for messages, so I don't think we need the "Message" postfix in the key name. We can just say: `smartTransactionsEnabled`